torch/_dynamo/variables/torch.py

```diff
@@ -401,8 +402,16 @@ def call_function(
 class TorchInGraphFunctionVariable(BaseTorchVariable):
     """Points to a torch function/method that should be put in FX graph"""
 
+    def __init__(self, value, traceable=None, **kwargs) -> None:
+        super().__init__(value, **kwargs)
+        from ..trace_rules import is_nonstrict_trace_callable
+
+        if traceable is None:
+            traceable = is_nonstrict_trace_callable(value)
+        self.traceable = traceable
```

Comment:
nit: call this nonstrict_traceable to avoid confusion

Reply:
Updated.